src/app/auth/containers/auth/auth.component.html

```diff
@@ -1,3 +1,10 @@
-<p>
-  auth works!
-</p>
+<div class="mat-form">
+    <div *ngIf='!signupVisible'>
+        <app-login (loginRequested)='login($event)'></app-login>
```

Comment:
maybe handleLogin instead of login (same for signUp), as then method name begins with a verb